src/qt/sendcoinsentry.cpp

```diff
@@ -44,6 +44,27 @@ void SendCoinsEntry::on_pasteButton_clicked()
     ui->payTo->setText(QApplication::clipboard()->text());
 }
 
+void SendCoinsEntry::on_fullamount_clicked(){
+    if(!model)
+        return;
+
+    ui->payAmount->setValue(model->getAvailableAmount(1));
+}
+
+void SendCoinsEntry::on_halfamount_clicked(){
```

Comment:
I believe that the project tries to follow the convention that { are placed on the line below the actual function name etc..
Even though I prefer the other way, but having things consistent is always nice.